src/client.js

```diff
@@ -2,17 +2,19 @@
 class Client {
   constructor(url) {
     this.url = url;
+    this.listLocations = this.listLocations.bind(this);
   }
 
   createConnectionToken() {
     const formData = new URLSearchParams();
     return this.doPost(this.url + "/connection_token", formData);
   }
 
-  registerDevice({ label, registrationCode }) {
+  registerDevice({ label, registrationCode, location }) {
     const formData = new URLSearchParams();
     formData.append("label", label);
     formData.append("registration_code", registrationCode);
+    formData.append("location", location);
     return this.doPost(this.url + "/register_reader", formData);
```

Comment:
did we update the `register_reader` endpoint for the location param?

Reply:
Ah good catch. Just added it in the other PR. https://github.com/stripe/example-terminal-backend/pull/32/commits/5f489d2bc6160412ca831ab7a7d9aab5340cd26f